Ginger/GingerCoreCommon/Repository/RepositoryFolder.cs

```diff
@@ -82,6 +82,11 @@ public string DisplayName
             }
         }
 
+        public override string ToString()
```

Comment:
please explain

Reply:
to display nice name when we drag the API Folder. Currently it displays complete class path as, "Amdocs.Ginger.Repository.RepositoryFolder".

Reply:
*Amdocs.Ginger.Repository.ApplicationModels